pkg/volume/nfs/nfs.go

```diff
@@ -17,22 +17,19 @@ limitations under the License.
 package nfs
 
 import (
-	"os"
-
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/types"
-	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/volume"
-	"github.com/golang/glog"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/volume/network_volume"
 )
 
 // This is the primary entrypoint for volume plugins.
 func ProbeVolumePlugins() []volume.VolumePlugin {
-	return []volume.VolumePlugin{&nfsPlugin{nil, newNFSMounter()}}
+	return []volume.VolumePlugin{&nfsPlugin{&network_volume.NetworkVolumePlugin{nfsPluginName, nil}, newNFSMounter()}}
```

Comment:
nit: I prefer explicitly stating the struct members that are being initialized! Since this change has to happen in a few other places in volumes, I am fine with making that change in a separate PR.